compiler/rustc_const_eval/src/interpret/validity.rs

```diff
@@ -547,6 +534,25 @@ impl<'rt, 'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> ValidityVisitor<'rt, 'mir, '
         Ok(())
     }
 
+    fn static_mutability(&self, mutability: Mutability, nested: bool, did: DefId) -> Mutability {
```

Comment:
If you make this take just `tcx` or `ecx` rather than `self`, you can avoid the new `unwrap` above.